README.md

```diff
@@ -2,145 +2,174 @@
 
 <br/>
 
-### Generate a Node.js app that is packed with best practices AND simplicty in mind. Based on our repo [Node.js best practices](https://github.com/goldbergyoni/nodebestpractices) (77,000 stars)
+### Generate a Node.js app that is packed with best practices AND simplicity in mind. Based off our repo [Node.js best practices](https://github.com/goldbergyoni/nodebestpractices) (77,000 stars)
 
-❣️ **Alpha stage:** It's a very premature work-in-progress, we're only kicking-off now, please revisit by June 2022🗓
+<br />
 
-![Discord](/docs/images/discord-logo.png) [Discord discussions](https://discord.gg/9Nrarr7p) | ![Twitter](/docs/images/twitter-icon.png) [Twitter](https://twitter.com/nodepractices)
+❣️ **Alpha stage:** Practica.js is a work-in-progress. We've only recently kicked off. Please revisit by June 2022 🗓
 
-# What in here for you in one paragraph
+**Contact us:** ![Discord](/docs/images/discord-logo.png) [Chat with us on Discord](https://discord.gg/9Nrarr7p) | ![Twitter](/docs/images/twitter-icon.png) [Send us a Tweet](https://twitter.com/nodepractices)
```

Comment:
The phrase "one paragraph" makes the point of how short it is, let's keep it?

Reply:
How does 'A One Paragraph Overview' sound?

Reply:
Yes, love it